docs/application/native/guides/connectivity/connection.md

```diff
@@ -154,6 +154,55 @@ To get the type of the current connection, IP address, and proxy information:
     }
     ```
 
+<a name="getting-internet-connection-state-information"></a>
```

Comment:
In case of "getting-internet-connection-state-information", you don't need to set like this.
It is automatically generated by md file when \## is used.
So please delete this line.